core/src/main/java/org/tiqr/core/authentication/AuthenticationSummaryFragment.kt

```diff
@@ -61,6 +61,8 @@ class AuthenticationSummaryFragment : BaseFragment<FragmentAuthenticationSummary
 
         binding.buttonOk.setOnClickListener {
             findNavController().popBackStack()
+            getActivity().finish();
```

Comment:
This is Java code and is not correct/compiling. 

If you want the app to close:
- remove the `findNavController().popBackStack()` line
- remove `getActivity().finish();`
- replace `System.exit(0);` with:
```
requireActivity().finishAffinity()
exitProcess(0)
```

Reply:
Thank you for your suggestions. I've implemented and tested the changes.